docs/DOCKER-MACHINE.md

```diff
@@ -0,0 +1,45 @@
+# Docker Machine
+
+Before the Docker native client we used docker-machine. Here are the steps for this original setup process.
+
+## Docker dependencies
+
+The project is managed via Docker containers. Right now we're using Docker
+version 1.9.0 with Docker Machine version 0.5.0.
+
+Commands below assume OS X and preference to install libraries via Homebrew.
```

Comment:
I think it's better to say that the best way to install any of these Docker tools it to follow the current Docker documentation on https://docker.github.io/engine/installation/

On top of that, installing Docker native installs Docker Machine as well so they're not exclusive approaches.  A link to the source seems more robust to me.